lib/Factory/MediaFactory.php

```diff
@@ -633,6 +633,10 @@ public function query($sortOrder = null, $filterBy = [])
             $params['type'] = $this->getSanitizer()->getString('type', $filterBy);
         }
 
+        if ($this->getSanitizer()->getInt('imageProcessing', $filterBy) !== null) {
+            $body .= 'AND ( media.type = \'image\' OR media.type = \'module\' ) ';
```

Comment:
we should also say where moduleSystemFile = 0 so that we don't process module files that are images (like the Flags in the stocks module for example)

something like:
```
$body .= 'AND ( media.type = \'image\' OR (media.type = \'module\' and `media`.moduleSystemFile =0) ) ';
```